src/test/java/net/masterthought/cucumber/generators/integrations/PageTest.java

```diff
@@ -38,4 +42,13 @@ protected DocumentAssertion documentFrom(String pageName) {
             throw new ValidationException(e);
         }
     }
+
+    protected String[] getMessages(Output[] outputs) {
+        List<String> messages = new ArrayList<>();
+        for (Output output : outputs) {
+            messages.addAll(Arrays.asList(output.getMessages()));
```

Comment:
This method used only in test and designed for getting output messages for step tests and feature report page integration tests.